### PR TITLE
Add top_5 to to eval to resnet

### PR DESCRIPTION
**Pull request by tfboyd**
Add top 5 accuracy to resnet eval

### CODE REVIEW

File: official/resnet/resnet_run_loop.py [hunk 1]
```diff
@@ -284,14 +284,15 @@ def exclude_batch_norm(name):
...
+      This function is used for fine tuning.
+
       Args:
-      gvs : list of tuples with gradients and variable info
+        gvs: list of tuples with gradients and variable info
       Returns:
-      filtered gradients so that only the dense layer remains
-      '''
+        filtered gradients so that only the dense layer remains
+      """
```
Comment (karmel) [inline official/resnet/resnet_run_loop.py]: Thanks

**Comment by karmel** [comment]
Looks like there are test failures with Cifar for the squeeze-- can you take a look?

**Comment by tfboyd** [comment]
Yup not sure why it would have a different shape but I am sure I will find out.

**Comment by tfboyd** [comment]
@robieta   Thank you.  :-)

**Pull request merged**